cirq/ops/parity_gates.py

```diff
@@ -159,6 +160,14 @@ class ZZPowGate(eigen_gate.EigenGate,
         where w = e^{i \pi t} and '.' means '0'.
     """
 
+    def _decompose_(self, qubits):
+        # See eigen_gate.EigenGate for more info about the global phase
+        phase = np.exp(1j * np.pi * self.global_shift * self.exponent)
+        yield GlobalPhaseOperation(coefficient=phase)
```

Comment:
Could you double check that putting the wrong global phase causes a test failure, before submitting?

Reply:
It definitely does.  That's why I was over at your desk complaining about global phase.  Turns out I just had the wrong phase that didn't take into account global_shift

Reply:
@Strilanc `assert_eigengate_implements_consistent_protocols` checks several global_shift values.